3/3.11/download_source

```diff
@@ -0,0 +1,146 @@
+#!/bin/bash -eu
```

Comment:
We can get checksum of each file under templates directory and try to find files with the same checksum, maybe also the same name. What do you think?